app/assets/javascripts/order.js

```diff
@@ -4,8 +4,12 @@ $(document).ready(function() {
   $("#new_order").submit(function(event) {
     var $form = $(this);
     $form.find("button").prop("disabled", true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if ($("#card_number").length) {
+      Stripe.card.createToken($form, stripeResponseHandler);
+      return false;
+  } else {
+      true;
```

Comment:
1. Why are you return false/true here? Not being critical, I just don't know what's going on
2. Gotta say `return true`. JavaStupid, amirite????